application/src/main/data/upgrade/3.6.2/schema_update_attribute_kv.sql

```diff
@@ -0,0 +1,181 @@
+--
+-- Copyright © 2016-2023 The Thingsboard Authors
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+
+-- create new attribute_kv table schema
+DO
+$$
+    BEGIN
+        -- in case of running the upgrade script a second time:
+        IF EXISTS(SELECT 1 FROM information_schema.columns WHERE table_name = 'attribute_kv' and column_name='entity_type') THEN
```

Comment:
Maybe we should leave entity type? But migrate it to ordinal as well.
@ashvayka What do you think?